client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -319,6 +324,47 @@ const getVisiblePlugins = ( plugins, country, industry, productTypes ) => {
 	);
 };
 
+const transformRemoteExtensions = ( extensionData, localeValue ) => {
+	return extensionData.reduce( ( result, extension ) => {
+		const localeData =
+			extension.locales.find(
+				( locale ) => locale.locale === localeValue
+			) ||
+			extension.locales.find( ( locale ) => locale.locale === 'en_US' );
+		const transformedExtension = {
+			title: localeData.title,
+			slug: extension.key,
+			description: generatePluginDescriptionWithLink(
+				localeData.description,
+				extension.key
+			),
+			isVisible: () => true,
+		};
+		const sectionIndex = extension.section === 'primary' ? 0 : 1;
+		result[ sectionIndex ].plugins.push( transformedExtension );
+		return result;
+	}, initialInstallableExtensions );
+};
+
+const baseValues = { install_extensions: true };
+const createInitialValues = ( extensions, country, industry, productTypes ) => {
+	return extensions.reduce( ( acc, curr ) => {
+		const plugins = getVisiblePlugins(
+			curr.plugins,
+			country,
+			industry,
+			productTypes
+		).reduce( ( pluginAcc, { slug } ) => {
+			return { ...pluginAcc, [ slug ]: true };
+		}, {} );
+
+		return {
+			...acc,
+			...plugins,
+		};
+	}, baseValues );
```

Comment:
Although this one references a static variable it should be fine here, as you return a new object within reduce method.